README.md

```diff
@@ -7,14 +7,14 @@ This is a work in progress. Idea is to treat this as scaffolding for future proj
 It will also be a learning of all these technologies in React Ecosystem.
 
 ### Technologies to be incorporated
-* React 16.3
-* TypeScript (Both on client and server side)
-* React Router 4
-* PostCSS for CSS
-* Webpack 4
-* Jest (React Testing)
-* Apollo
-* GraphQL
-* Express 
-* MongoDB
-* Testing for Express (not sure which framework to use)
+* [React 16.3](https://reactjs.org/)
+* [TypeScript (Both on client and server side)](https://www.typescriptlang.org/)
+* [React Router 4]()](https://reacttraining.com/react-router/)
```

Comment:
this is breaking. pls check in preview mode.

Reply:
I made the changes. Works fine. Please check